fe/fe-core/src/main/java/org/apache/doris/analysis/DataDescription.java

```diff
@@ -195,6 +195,8 @@ public DataDescription(String tableName,
         this.deleteCondition = deleteCondition;
         this.sequenceCol = sequenceColName;
         this.properties = properties;
+        columnsNameToLowerCase(fileFieldNames);
```

Comment:
if this change take effect, would the column name of query result become low case?

Reply:
No. This change only change the user input stream/broker load columns name, will not affect the table columns.